geospaas_harvesting/ingesters.py

```diff
@@ -387,6 +388,20 @@ def _get_normalized_attributes(self, url, *args, **kwargs):
         return normalized_attributes
 
 
+class ThreddsIngester(DDXIngester):
+    """Ingest datasets from the DDX metadata of the OpenDAP service of a Thredds server"""
+
+    url_matcher = re.compile(r'^(.*)/(fileServer)/(.*)$')
+
+    @classmethod
+    def prepare_url(cls, url):
+        url_match = cls.url_matcher.match(url)
```

Comment:
Is it possible to do it in a simpler way (instead of above mentioned code and by removing the `url_matcher`  attribute)?
```python

    @ staticmethod
    def prepare_url(url):
        url_match = re.compile(r'^(.*)/(fileServer)/(.*)$').match(url)
```
When there is no need to have a classmethod, we can do it by a static method instead in a simpler way. right?

Reply:
If you do it that way, the regex is compiled for every URL.
The way I did it, the regex is compiled once.

Reply:
So you do that because of increasing the speed and performance of code?

Reply:
Yes